luminaire/model/model_utils.py

```diff
@@ -79,6 +82,8 @@ class LADHolidays(pandas.tseries.holiday.AbstractHolidayCalendar):
 
     NewYearsDay = _H.Holiday('New Years Day', month=1, day=1)
     NewYearsDayObserved = _H.Holiday('New Years Day Observed', month=1, day=1, observance=_H.nearest_workday)
+    Jan2 = _H.Holiday('January 2', month=1, day=2)
```

Comment:
Nit: Something similar like `DayAfterNewYear`

Reply:
Same as above. This is just another cushion after the holiday.

Reply:
yes, but do we want the variable name to be MonthDate like `Jan2 `? I was proposing to use some user-friendly variable names for such thing like `DayAfterNewYear `

Reply:
Please see the comment above.